docs/content/Deployment/Production-Checklist.md

```diff
@@ -144,3 +144,12 @@ server {
   }
 }
 ```
+
+## Set up health checks
```

Comment:
@hassankhan I think there's more context to these conventional API methods :) AFAIK they come from k8s world, right? Maybe we can have a relevant link here? Also, if there's a popular observability tool (Prometheus? Something AWS/GCP-specific?) which has a guide on working with these endpoints, can we reference it, too?

Reply:
Definitely agree on referencing the K8S docs, not too sure about promoting another tool but perhaps we can address that in a future PR?